Machine.Specifications.ReSharper.Provider.10/MspecServiceProvider.cs

```diff
@@ -28,7 +28,9 @@ public class MspecServiceProvider
             IUnitTestElementManager elementManager,
             IUnitTestElementIdFactory elementIdFactory,
             IUnitTestElementCategoryFactory categoryFactory,
-            IDotNetCoreSdkResolver dotNetCoreSdkResolver)
+            IDotNetCoreSdkResolver dotNetCoreSdkResolver,
+            IUnitTestAgentManager agentManager,
+            IUnitTestResultManager resultManager)
```

Comment:
If you make `MspecOutOfProcessUnitTestRunStrategy` injectable (using `SolutionComponent` as per other comment), then you can just inject `MspecOutOfProcessUnitTestRunStrategy` here instead, no need for all these other interfaces.